src/webots/wren/WbWrenRenderingContext.hpp

```diff
@@ -38,6 +38,8 @@ class WbWrenRenderingContext : public QObject {
     VF_OUTLINE = 0x00000001,  // flag for bounding objects, axes and centers of mass, i.e. outlines that should not be rendered
                               // by a WbCamera
     VF_SELECTED_OUTLINE = 0x00000002,  // flag for selected outlines
+    VF_BILLBOARDS = 0x00000003,        // flag for billboards
```

Comment:
That looks weird, as these flags seems to be aimed at being ORed (they are all powers of 2).
Introducing VF_BILLBOARDS this way may cause unwanted effects, as it equals VF_OUTLINE | VF_SELECTED_OUTLINE.